gitlabform/lists/projects.py

```diff
@@ -113,6 +117,39 @@ def _get_projects(self, target: str, groups: Groups) -> Projects:
                 archived_projects_from_configuration_not_from_groups,
             )
 
+        if target == "ALL":
+            # in this case we also need to get the list of projects being transfered
+            # to a different namespace from the config
+
+            projects_from_configuration = self.configuration.get_projects()
+
+            for project in projects_from_configuration:
+                try:
+                    project_transfer_source = self.configuration.config[
+                        "projects_and_groups"
+                    ][project]["project"]["transfer_from"]
+                except KeyError:
+                    # ignore projects that don't contain transfer_from key
+                    continue
```

Comment:
Correct me if I'm wrong. I think we can use either `continue` or `pass`, right? I think everywhere else in this repo, `pass` is being used. Maybe follow the pattern for consistency?

Reply:
I've done some refactoring to reduce code duplication, and this is no longer used.

Reply:
Thanks. Let's close this thread. Will open new discussion if needed.